Python/pytime.c

```diff
@@ -41,6 +40,14 @@
 #  error "unsupported time_t size"
 #endif
 
+#if PY_TIME_T_MAX + PY_TIME_T_MIN != -1
```

Comment:
It would be quite extraordinary if either of these failed; I'll remove these checks once the buildbots have run to completion.

Reply:
There's no harm in leaving them in, it serves as documentation of assumptions that later code makes. (agreed though. I'd be surprised to find any other number format on a mainstream cpu ever again)

Reply:
Fair enough; leaving them in.